sqld/src/rpc/wal_log.rs

```diff
@@ -90,11 +90,11 @@ impl WalLog for WalLogService {
             guard.insert(replica_addr);
         }
         let response = HelloResponse {
-            database_id: self.logger.database_id.to_string(),
+            database_id: dbg!(self.logger.database_id.to_string()),
```

Comment:
We probably want this `dbg!(` removed :)

Reply:
they're gone later